editor/src/components/mouse-move.ts

```diff
@@ -13,8 +13,21 @@ export function mouseWheelHandled() {
   didWeHandleWheelForThisFrame = true
```

Comment:
hmmm, I wonder if we can now safely delete this whole chrome workaround... we should create a ticket to check if it's still needed

Reply:
I made a ticket